pandas/tests/reshape/test_pivot.py

```diff
@@ -301,6 +301,34 @@ def test_pivot_multi_functions(self):
         expected = concat([means, stds], keys=['mean', 'std'], axis=1)
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_multiple_columns_as_index(self):
+        # adding the test case for multiple columns as index (#21425)
+        df = DataFrame({'lev1': [1, 1, 1, 1, 2, 2, 2, 2],
+                        'lev2': [1, 1, 2, 2, 1, 1, 2, 2],
+                        'lev3': [1, 2, 1, 2, 1, 2, 1, 2],
+                        'values': [0, 1, 2, 3, 4, 5, 6, 7]})
+        result = df.pivot(index=['lev1', 'lev2'],
+                          columns='lev3',
+                          values='values')
+        result_no_values = df.pivot(index=['lev1', 'lev2'],
+                                    columns='lev3')
+        data = [[0, 1], [2, 3], [4, 5], [6, 7]]
```

Comment:
can you do the test caess 1 after the other and use the standard

result=
expected=
assert_frame_equal(result, expected)